docs/book/src/quick-start.md

```diff
@@ -24,64 +32,67 @@ sudo mv /tmp/kubebuilder_2.0.1_${os}_${arch} /usr/local/kubebuilder
 export PATH=$PATH:/usr/local/kubebuilder/bin
 ```
 
-You can also install a KubeBuilder master snapshot from
-`https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
-
-Install [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
+**Note:** Also, you can install a master snapshot from `https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
 
 ## Create a Project
 
-Initialize a new project and Go module for your controllers:
+Create a directory then run the following command inside of it to initialize a new project:
 
 ```bash
 kubebuilder init --domain my.domain
 ```
 
-If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
+**Note:** If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
 in order to tell kubebuilder and Go the base import path of your module.
 
 ## Create an API
 
-Create a new API group-version called `webapp/v1`, and a kind `Guestbook`
-in that API group-version:
+* Run the following command to create a new API (group/version) as `webapp/v1` and the new Kind(CRD) `Guestbook` on it:
 
 ```bash
 kubebuilder create api --group webapp --version v1 --kind Guestbook
 ```
 
-This will create the files `api/v1/guestbook_types.go` and
-`controller/guestbook_controller.go` for you to edit.
+**Note:** If you press `y` for Create Resource [y/n] and for Create Controller [y/n] then this will create the files `api/v1/guestbook_types.go` where the API is defined 
+and the `controller/guestbook_controller.go` where the reconciliation business logic is implemented for this Kind(CRD) if you choose yes for the both options.  
 
-**Optional:** Edit the API definition or the reconciliation business
-logic. For more on this see [What's in
-a Controller](cronjob-tutorial/controller-overview.md) and [Designing an
-API](/cronjob-tutorial/api-design.md).
+* **OPTIONAL:** Edit the API definition and the reconciliation business
+logic. For more info see [Designing an API](/cronjob-tutorial/api-design.md) and [What's in
+a Controller](cronjob-tutorial/controller-overview.md). Following an example. 
+
+```go (api/v1/guestbook_types.go)
+// GuestbookSpec defines the desired state of Guestbook
+type GuestbookSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+    // Quantity of instances
+    // +kubebuilder:validation:Minimum=1
+    Size int32 `json:"size"`
+}
+```
 
-## Test It Out Locally
+## Test It Out 
 
 You'll need a Kubernetes cluster to run against.  You can use
 [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or
 run against a remote cluster.
 
-Your controller will automatically use the current context in your
-kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
-
-Install the CRDs into the cluster:
-
+* Install the CRDs into the cluster:
```

Comment:
note that if you're adding bullet points, you should make sure the fenced code block is indented to line up with the start of the text so that the code block joins the list item properly.

Reply:
Ditto everywhere else in this doc

Reply:
👍